invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -293,9 +294,8 @@ def dashboard(dashboard_name=None):
     )
 
 
-@login_required
-def deposit_create():
-    """Create a new deposit."""
+@pass_draft_community
```

Comment:
did you remove the `login_required` because it is already checked in the service?

Reply:
oups, unintended. THanks!